### PR TITLE
Fix BCD in yaml for RTC*Stats.* that have a page and an entry

### PR DESCRIPTION
**Pull request by teoli2003**
We have many RTC*Stats.* pages with broken specs and compat tables (red tables).

This PR fixes the cases where we actually have entries in mdn/browser-compat-table, but we weren't linking to the right places.

I also fixed a few obvious orthographic or grammar errors.

### CODE REVIEW

File: files/en-us/web/api/rtcinboundrtpstreamstats/nackcount/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "RTCInboundRtpStreamStats: nackCount property"
 short-title: nackCount
 slug: Web/API/RTCInboundRtpStreamStats/nackCount
 page-type: web-api-instance-property
-browser-compat: api.RTCInboundRtpStreamStats.nackCount
+browser-compat: api.RTCStatsReport.type_inbound-rtp.nackCount
```
Comment (OnkarRuikar) [inline files/en-us/web/api/rtcinboundrtpstreamstats/nackcount/index.md:6]: Shouldn't this be `deprecated` or `experimental` in BCD? No browser seems to support this.

Comment (teoli2003) [reply files/en-us/web/api/rtcinboundrtpstreamstats/nackcount/index.md:6]: I guess it is deprecated: 
Capture d’écran 2023-12-19 à 08 08 28

**Comment by hamishwillee** [comment]
I will look at this tomorrow.

**Review by hamishwillee — APPROVED** [review]
Excellent

**Pull request merged**